backend/app/Chat/Redeem/RedeemCoins.php

```
@@ -240,4 +239,119 @@ public static function get(int $id): ?array
+                $dbConn->rollBack();
+                return false;
+            }
+
+            // Check if code has uses left
+            if ((int) $codeData['uses'] <= 0) {
+                $dbConn->rollBack();
+                return false;
+            }
+
+            // Check if user has already redeemed this code
+            $stmt = $dbConn->prepare('SELECT COUNT(*) FROM ' . RedeemRedeems::getTableName() . ' WHERE code = :code AND user = :user AND deleted = "false" FOR UPDATE');
```

PHPMD (2.15.0)

Avoid using static access to class 'MythicalDash\Chat\Redeem\RedeemRedeems' in method 'redeemCodeAtomic'. (Clean Code Rules)

(StaticAccess)